bin/install_uninstall.ml

```diff
@@ -207,18 +207,49 @@ let install_uninstall ~what =
        | _ -> ());
       let module CMap = Map.Make(Context) in
       let install_files_by_context =
-        CMap.of_list_multi install_files |> CMap.to_list
+        CMap.of_list_multi install_files
+        |> CMap.to_list
+        |> List.map ~f:(fun (context, install_files) ->
+          let entries_per_package =
+            Package.Name.Map.of_list_map_exn install_files
+              ~f:(fun (package, install_file) ->
+                let entries = Install.load_install_file install_file in
+                match
+                  List.filter_map entries ~f:(fun entry ->
+                    Option.some_if
+                      (not (Path.exists (Path.build entry.src)))
+                      entry.src)
+                with
+                | [] -> (package, entries)
+                | missing_files ->
+                  let pp =
+                    let open Pp in
+                    List.map missing_files ~f:(fun p ->
+                      concat
+                        [ verbatim "- "
```

Comment:
Other places use `Fmt.prefix`, I think that could work here

Reply:
I think we are moving away from `Fmt` in favor of `Pp`.